book/tutorials/core-datasets/03_practice-querying.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "id": "minimal-cuisine",
+   "id": "urban-steal",
```

Comment:
I wish that the ID's would not be renamed every time someone opens a notebook - maybe some day ....

Reply:
yes, this is actually due to a bug that was fixed for jlab>=3.0.13 (https://github.com/jupyterlab/jupyterlab/pull/10018) https://github.com/jupyterlab/jupyterlab/releases/tag/v3.0.13